This patch make Xfce use "elementary-xfce-dark" as the default icon theme, "Greybird" as
the default GTK theme and enable font antialias.

--- xfce4-settings-4.20.1/xfsettingsd/xsettings.xml.orig	2024-12-20 09:53:09.518493612 +0800
+++ xfce4-settings-4.20.1/xfsettingsd/xsettings.xml	2024-12-20 09:55:15.506496473 +0800
@@ -8,4 +8,4 @@
   <property name="Net" type="empty">
-    <property name="ThemeName" type="string" value="Adwaita"/>
-    <property name="IconThemeName" type="string" value="Adwaita"/>
+    <property name="ThemeName" type="string" value="Greybird"/>
+    <property name="IconThemeName" type="string" value="elementary-xfce-dark"/>
     <property name="DoubleClickTime" type="int" value="400"/>
@@ -21,5 +21,5 @@
     <property name="DPI" type="int" value="96"/>
-    <property name="Antialias" type="int" value="-1"/>
+    <property name="Antialias" type="int" value="1"/>
     <property name="Hinting" type="int" value="-1"/>
-    <property name="HintStyle" type="string" value="hintnone"/>
+    <property name="HintStyle" type="string" value="hintfull"/>
     <property name="RGBA" type="string" value="none"/>
